FWCore/MessageService/test/UnitTestClient_H.cc

```diff
@@ -14,8 +14,17 @@ void
                            , edm::EventSetup const & /*unused*/
                               )
 {
+#ifdef EDM_ML_DEBUG
```

Comment:
I wasn't really sure how the MessageLogger unit tests are crafted. To me it looked like that `UnitTestClient_H.cc` would be the version compiled without `EDM_ML_DEBUG`, and `UnitTestClient_Hd.cc` would be the same but with `EDM_ML_DEBUG` (it is defined in the first line).

(if this is the case, maybe `UnitTestClient_Hd.cc` could be replaced simply by
```
 #define EDM_ML_DEBUG
#include "UnitTestClient_H.cc"
```
to avoid copy-paste?)

If this is not the case, I'll need to play around a bit more (`scram b runtests` succeeds for the modified unit tests, I thought that would be sufficient)